### ISSUE TITLE
Presto doesn't start on local machine with Java 8

### ISSUE THREAD

**Issue by sopel39** [prestodb/presto#9592]
When starting newest version of Presto from `master` from IntelliJ with Java 8 the following error happens:
```
2017-12-20T15:55:59.785+0100	WARN	query-management-0	com.facebook.presto.memory.RemoteNodeMemory	Memory info update request to http://172.20.1.10:8080/v1/memory has not returned in 21.94s
Disconnected from the target VM, address: '127.0.0.1:65173', transport: 'socket'
2017-12-20T15:56:28.410+0100	ERROR	main	com.facebook.presto.server.PrestoServer	Error loading pom: /Users/sopel39/Desktop/Projekty/Starburstdata/presto/presto-main/../presto-blackhole/pom.xml
java.lang.RuntimeException: Error loading pom: /Users/sopel39/Desktop/Projekty/Starburstdata/presto/presto-main/../presto-blackhole/pom.xml
	at io.airlift.resolver.ArtifactResolver.resolvePom(ArtifactResolver.java:141)
	at com.facebook.presto.server.PluginManager.buildClassLoaderFromPom(PluginManager.java:239)
	at com.facebook.presto.server.PluginManager.buildClassLoader(PluginManager.java:228)
	at com.facebook.presto.server.PluginManager.loadPlugin(PluginManager.java:152)
	at com.facebook.presto.server.PluginManager.loadPlugins(PluginManager.java:140)
	at com.facebook.presto.server.PrestoServer.run(PrestoServer.java:116)
	at com.facebook.presto.server.PrestoServer.main(PrestoServer.java:66)
Caused by: org.apache.maven.project.ProjectBuildingException: Some problems were encountered while processing the POMs:
[ERROR] Failed to determine Java version for profile java9-surefire @ io.airlift:airbase:78, /Users/sopel39/.m2/repository/io/airlift/airbase/78/airbase-78.pom, line 1324, column 22

	at org.apache.maven.project.DefaultProjectBuilder.build(DefaultProjectBuilder.java:176)
	at org.apache.maven.project.DefaultProjectBuilder.build(DefaultProjectBuilder.java:102)
	at io.airlift.resolver.ArtifactResolver.resolvePom(ArtifactResolver.java:134)
	... 6 more
Caused by: org.apache.maven.model.building.ModelBuildingException: 1 problem was encountered while building the effective model for com.facebook.presto:presto-blackhole:0.192-SNAPSHOT
[ERROR] Failed to determine Java version for profile java9-surefire @ io.airlift:airbase:78, /Users/sopel39/.m2/repository/io/airlift/airbase/78/airbase-78.pom, line 1324, column 22

	at org.apache.maven.model.building.DefaultModelProblemCollector.newModelBuildingException(DefaultModelProblemCollector.java:195)
	at org.apache.maven.model.building.DefaultModelBuilder.build(DefaultModelBuilder.java:419)
	at org.apache.maven.model.building.DefaultModelBuilder.build(DefaultModelBuilder.java:371)
	at org.apache.maven.model.building.DefaultModelBuilder.build(DefaultModelBuilder.java:362)
	at org.apache.maven.model.building.DefaultModelBuilder.build(DefaultModelBuilder.java:232)
	at org.apache.maven.project.DefaultProjectBuilder.build(DefaultProjectBuilder.java:142)
	... 8 more

```

I think this is related to: https://github.com/airlift/airbase/commit/87a1977311179f6e4097768a7ea3336d11298a9f

```
bash-3.2$ java -version
java version "1.8.0_152"
```

CC: @electrum @ilfrin @findepi

**Comment by findepi** [comment]
@electrum this would fix the issue: https://github.com/airlift/resolver/pull/14

**Comment by electrum** [comment]
I’m working on this.

**Comment by electrum** [comment]
@findepi Thanks for figuring that out! I released resolver: #9593
